CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ Making a good PR or Issue
 -------------------------
 
 Since Github _PRs_ and _Issues_ are now our primary way of tracking both code
-changes and outstanding issues, it's important the we wall play nicely. Here
+changes and outstanding issues, it's important that we wall play nicely. Here
```

Comment:
... we `wall`? another typo?

Reply:
I'm not a native English speaker. I examined the sentence with a commercial grammar checker, but it said there is nothing wrong with the word `wall`, so I guess it may have another meaning. However, It is OK for me to change it.

Reply:
Wall in this caseshould be replaced with all.

Reply:
Done.